scripts/postinstall.js

```diff
@@ -8,26 +8,37 @@ const isWindows = process.platform === 'win32';
 
 const truthyStr = (val) => val && !['0', 'false', 'f', 'n', 'no'].includes(val.toLowerCase());
 const { CI, ADBLOCK, SILENT } = process.env;
-if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
-  const messageTokens = ['Serverless Framework successfully installed!'];
-  if (isStandaloneExecutable && !isWindows) {
-    messageTokens.push(
-      'To start your first project, please open another terminal and run “serverless”.',
-      'You can uninstall at anytime by running “serverless uninstall”.'
+const isNpmGlobalPackage = require('../lib/utils/npmPackage/isGlobal');
+
+(async () => {
+  if (!truthyStr(CI) && !truthyStr(ADBLOCK) && !truthyStr(SILENT)) {
+    const messageTokens = ['Serverless Framework successfully installed!'];
+
+    if ((isStandaloneExecutable && !isWindows) || (await isNpmGlobalPackage())) {
+      messageTokens.push(
+        'You can turn on automatic updates by running "serverless config --autoupdate".'
```

Comment:
I've just double checked the order of messages, and I think that in case of standalone, this message should go somewhere aside `serverless install`.

As configured currently, it'll show as:

```
You can turn on automatic updates by running "serverless config --autoupdate".

To start your first project, please open another terminal and run “serverless”

You can uninstall at anytime by running “serverless uninstall”.'
```

And that doesn't seem to look good. I'd see order rather as:

```
To start your first project, please open another terminal and run “serverless”

You can turn on automatic updates by running "serverless config --autoupdate".

You can uninstall at anytime by running “serverless uninstall”.'
```

And, now when looking at this message, I'll improve wording so it's not as repetitive:

```
To start your first project, please open another terminal and run “serverless”

Turn on automatic updates with "serverless config --autoupdate".

Uninstall at anytime with “serverless uninstall”.'
```

(I'll adapt same rewording to non standalone version)

What do you think?

Reply:
That is a great point - I've adapted the messages and ensured better ordering. Please let me know what do you think :bow: